doc/source/ray-more-libs/joblib.rst

```diff
@@ -21,13 +20,25 @@ that use scikit-learn from a single node to a cluster.
 Quickstart
 ----------
 
-To get started, first `install Ray <installation.html>`__, then use
-``from ray.util.joblib import register_ray`` and run ``register_ray()``.
-This will register Ray as a joblib backend for scikit-learn to use.
-Then run your original scikit-learn code inside
-``with joblib.parallel_backend('ray')``. This will start a local Ray cluster.
-See the `Run on a Cluster`_ section below for instructions to run on
-a multi-node Ray cluster instead.
+`Install Ray <../ray-overview/installation.html>`__ and call some methods in your scikit-learn code as the following.
+
+.. code-block:: python
+
+  import joblib
+  from ray.util.joblib import register_ray
+  register_ray()
+  with joblib.parallel_backend('ray'):
+      # scikit-learn code
+
+Usage
+-----
+
+``from ray.util.joblib import register_ray`` imports joblib with Ray backend library.
+``register_ray()`` registers Ray as a joblib backend for scikit-learn to use.
```

Comment:
```suggestion
``register_ray()`` registers Ray as the joblib backend for scikit-learn to use.
```

Reply:
also, you can use backticks to format the library names like: `joblib` and `scikit-learn`